tools/docker-compose-datasets-server.yml

```diff
@@ -237,7 +228,7 @@ services:
       PARQUET_AND_INFO_URL_TEMPLATE: ${PARQUET_AND_INFO_URL_TEMPLATE-/datasets/%s/resolve/%s/%s}
       PARQUET_METADATA_STORAGE_DIRECTORY: ${PARQUET_METADATA_STORAGE_DIRECTORY-/parquet_metadata}
       ROWS_INDEX_MAX_ARROW_DATA_IN_MEMORY: ${ROWS_INDEX_MAX_ARROW_DATA_IN_MEMORY-300_000_000}
-      WORKER_STORAGE_PATHS: ${ASSETS_STORAGE_DIRECTORY-/assets}
+      WORKER_STORAGE_PATHS: ${ASSETS_STORAGE_ROOT-/storage/assets}
```

Comment:
Do we use this variable somewhere?

Reply:
we will remove it here: https://github.com/huggingface/datasets-server/pull/2286, but I made https://github.com/huggingface/datasets-server/pull/2286 depend on this one, so...